app/src/main/java/com/github/sdpcoachme/repositories/RepositoryAccess.kt

```diff
@@ -0,0 +1,18 @@
+package com.github.sdpcoachme.repositories
+
+import com.github.sdpcoachme.network.DataFormat
+
+import retrofit2.Response
+
+interface RepositoryAccess {
+
+    suspend fun getContentDB() : String
```

Comment:
As both what you call DB and what you call API are actually databases (the first is the "cache", and the second is the remote database), it might be better to rename them to something more distinct (I suggest cache for the local cache and remoteDB for the remote database, or something along those lines)